apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktRoute.kt

```diff
@@ -155,6 +155,19 @@ internal fun Route.aktivitetspliktRoutes(aktivitetspliktService: Aktivitetsplikt
             }
         }
 
+        route("statistikk/{$BEHANDLINGID_CALL_PARAMETER}") {
+            get {
+                kunSystembruker {
+                    logger.info("Henter aktivitetspliktDto for statistikk")
+                    val dto =
+                        inTransaction {
+                            runBlocking { aktivitetspliktService.hentAktivitetspliktDto(sakId, brukerTokenInfo, behandlingId) }
```

Comment:
runBlocking og suspend om hverandre her, kanskje ikke helt det vi ønsker.

Reply:
Ja, det er kanskje verdt å se over, men tar ut denne fiksen for å ta bort denne feilen fra loggene :D